developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst

```diff
@@ -143,7 +143,7 @@ PHP 5.6 (supported)
 
     $ sudo apt-get update
     $ sudo apt-get install mongodb
-    $ sudo apt-get install php5-mongo
+    $ sudo apt-get install php-mongo
```

Comment:
In this case, you are in the "php 5.6" section.

How about add this instructions in the "PHP 7.0 (experimental)" section?

````
sudo apt-get remove php5-mongo
sudo apt-get install php-mongo
````